scripts/variantstore/README.md

```diff
@@ -1,6 +1,70 @@
-## Running the VAT pipeline
-The VAT pipeline is a set of WDLs
- - GvsSitesOnlyVCF.wdl
- - GvsValidateVAT.wdl
+# Getting started with GvsJointVariantCalling.wdl - Joint Calling with the Broad Genomic Variant Store
```

Comment:
Should this be in addition to the workspace Dashboard, or should they mirror one another? I ask because I think we'll want a lot of this info on the Dashboard.